pythonforandroid/toolchain.py

```diff
@@ -494,6 +494,10 @@ def add_parser(subparsers, *args, **kwargs):
         # However, it is also needed before the distribution is finally
         # assembled for locating the setup.py / other build systems, which
         # is why we also add it here:
+        parser_apk.add_argument(
```

Comment:
Could you cover test your change in `tests/test_toolchain.py`?
There're already some examples in this file and as I understand what you're doing in `toolchain.py` unit testing the behaviour should be fairly easy